embulk-input-mysql/src/test/java/org/embulk/input/mysql/BasicTest.java

```diff
@@ -41,6 +42,7 @@ private static String readResource(String fileName)
     @Before
     public void setup()
     {
+        TimeZone.setDefault(TimeZone.getTimeZone("Europe/Helsinki")); // +200
```

Comment:
Right. This should be unnecessary because gradle sets it. #94 and #95 should also don't need this line. Other than that, LGTM 👍